src/app/cube/new-home/learning-object-info/collections/collections.component.ts

```diff
@@ -7,9 +8,27 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CollectionsComponent implements OnInit {
 
-  constructor() { }
+  collections = [];
+  show = ['nccp', 'ncyte', 'intro_to_cyber'];
+
+  constructor(
+    private collectionService: CollectionService
+    ) { }
+
+  async ngOnInit(): Promise<void> {
+    //get collections
+    await this.collectionService
+    .getCollections()
+    .then((collections: Collection[]) => {
+      this.collections = collections;
+    })
+    .catch(e => {
+      console.error(e.message);
```

Comment:
This won't console in production. We can toast the message here. Something generic like "Error! There was a problem fetching collections. Please try again later"

Reply:
+1